README.md

```diff
@@ -1,2 +1,16 @@
-# TeamCity-TestFlight-Appium-ReactNative
-Guidance on integrating Kobiton service into the mobile app build pipeline: TeamCity, TestFlight, Appium and ReactNative.
+# Integrating Kobiton in TeamCity - HockeyApp - Appium - ReactNative development pipeline
+Guidance on integrating Kobiton service into the mobile app build pipeline: TeamCity, HockeyApp, Appium and ReactNative.
+
+If you are using:
++ TeamCity to build app
++ HockeyApp for keeping the daily app build
++ Appium to write the automation test
++ Reactnative to write app
+
+Kobiton is a mobile cloud platform that enables users to perform manual or automated testing on iOS and Android devices. This guide will demonstrate how to configure TeamCity to get the latest build on HockeyApp and run Appium automation tests on Kobiton Test Cloud service.
+
+Below is the diagram demonstrating how Kobiton integrates with your app development pipeline.
+
+![diagram](./Readme/assets/diagram.png)
+
+This guide will demonstrate [how to run automation tests on your app with the Kobiton service](./Readme/automation.md).
```

Comment:
`how to run automation tests on your app with the Kobiton service`, are you sure ?